src/pretix/base/payment.py

```diff
@@ -168,13 +168,22 @@ def identifier(self) -> str:
     @property
     def abort_pending_allowed(self) -> bool:
         """
-        Whether or not a user can abort a payment in pending start to switch to another
+        Whether or not a user can abort a payment in pending state to switch to another
         payment method. This returns ``False`` by default which is no guarantee that
         aborting a pending payment can never happen, it just hides the frontend button
         to avoid users accidentally committing double payments.
         """
         return False
 
+    @property
+    def requires_invoice_immediately(self):
+        """
+        Return whether this payment method requires an invoice to exist for an order, even though the event
+        is configured to only create invoices for paid orders.
```

Comment:
As an additional help for plugin authors, we could clarify here that the invoice exists before execute_payment is called